### PR TITLE
[fix] Fix lychee workflow

### PR DESCRIPTION
**Pull request by shreyan-gupta**
This was getting irritating...

### CODE REVIEW

File: lychee.toml [hunk 1]
```diff
@@ -16,4 +16,8 @@ exclude = [
 
     # StackOverflow bans GHA runners
     "^https://stackoverflow.com/",
+
+    # GNU websites exist but fail with "Too Many Requests"
```
Comment (Longarithm) [inline lychee.toml:20]: > exist but fail

mood.

File: lychee.toml [hunk 2]
```diff
@@ -16,4 +16,8 @@ exclude = [
...
+    "^http://www.gnu.org/licenses/"
+    "^https://www.gnu.org/licenses/why-not-lgpl.en.html"
```
Comment (Longarithm) [inline lychee.toml:22]:
```suggestion
    "^http://www.gnu.org/licenses/",
    "^https://www.gnu.org/licenses/why-not-lgpl.en.html",
```
?

**Review by Longarithm — APPROVED** [review]

File: lychee.toml (hunk 2 above)
Comment (shreyan-gupta) [reply lychee.toml:22]: aah crap...

File: lychee.toml (hunk 1 above)
Comment (shreyan-gupta) [reply lychee.toml:20]: 😂

**Pull request merged**